src/api/groupByService.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react'
+import { BASE_PATH } from './apiConfig'
+import agencyList from 'open-bus-stride-client/agencies/agencyList'
+
+/*
+example response
+[
+  {
+    "gtfs_route_date": "2023-02-27",
+    "operator_ref": 0,
+    "day_of_week": "string",
+    "total_routes": 0,
+    "total_planned_rides": 0,
+    "total_actual_rides": 0
+  }
+]
+*/
+export type GroupByResponse = {
+  gtfs_route_date: string
+  operator_ref: number
+  day_of_week: string
+  total_routes: number
+  total_planned_rides: number
+  total_actual_rides: number
+}[]
+
+async function groupby({
+  dateTo,
+  dateFrom,
+  groupBy,
+}: {
+  dateTo: string
+  dateFrom: string
+  groupBy: 'gtfs_route_date' | 'operator_ref' | 'day_of_week'
+}): Promise<GroupByResponse> {
+  // example: https://open-bus-stride-api.hasadna.org.il/gtfs_rides_agg/group_by?date_from=2023-01-27&date_to=2023-01-29&group_by=operator_ref
+  return (
+    await fetch(
+      `${BASE_PATH}/gtfs_rides_agg/group_by?date_from=${dateFrom}&date_to=${dateTo}&group_by=${groupBy}`,
```

Comment:
why not use the generated api? is it outdated? we can update

Reply:
I don't know how to use it, or how to regenerate the API configuration

Reply:
I re-generated the code https://github.com/ReallyLiri/open-bus-map-search/commit/fd1a54ece09d2f56d71b79e8522a491c5893f3cb
the npm package is https://www.npmjs.com/package/open-bus-stride-client
on how to use it - just take a look at `gtfsService.ts` and `siriService.ts`, should be pretty simple :)

Reply:
its already published